index.d.ts

```diff
@@ -80,6 +80,7 @@ declare module "http" {
     interface IncomingMessage {
       id: ReqId;
       log: pino.Logger;
+      allLogs: pino.Logger[];
```

Comment:
when multiple pino-http middleware are registered, the first registered middleware's pino logger will be attached to the req.log property. The remaining middleware's pino loggers (as well as the first) will be available in this new allLogs property.